ckanext/versioning/datapackage.py

```diff
@@ -0,0 +1,29 @@
+"""Frictionless Data datapackage.json related functions
+
+See http://specs.frictionlessdata.io/data-package/ for datapackage specs
+"""
+import frictionless_ckan_mapper.ckan_to_frictionless as ctf
+import frictionless_ckan_mapper.frictionless_to_ckan as ftc
+
+
+def dataset_to_frictionless(package):
+    """Convert a CKAN dataset dict to a Frictionless datapackage
+    """
+    # TODO: Use ckan_mapper
+    #return ctf.dataset(package)
+    return package
+
+
+def frictionless_to_dataset(package):
```

Comment:
As a small aside i'm increasingly inclining to using `dataset` rather than `package` everywhere and as a dataset name.